host/xtest/xtest_uuid_helpers.c

```diff
@@ -5,12 +5,15 @@
 
 #include <ctype.h>
 #include <endian.h>
+#ifdef OPENSSL_FOUND
 #include <openssl/evp.h>
+#endif
 #include <stdint.h>
 #include <string.h>
 #include <tee_api_types.h>
 #include <tee_client_api.h>
 #include "xtest_uuid_helpers.h"
+#include <adbg.h>
```

Comment:
This might not be needed.

Reply:
Right. This was from a overwritten commit. Will remove.